app/src/main/java/org/mozilla/fenix/search/toolbar/IncreaseDpsAction.kt

```diff
@@ -0,0 +1,20 @@
+package org.mozilla.fenix.search.toolbar
+
+import android.view.View
+import mozilla.components.concept.toolbar.Toolbar
+import org.mozilla.fenix.ext.increaseTapArea
+
+class IncreaseDpsAction(
```

Comment:
Nice approach 👍.
This seems to be a decorator, so could we name it as such to have the intent clearer? Something like `IncreasedTapAreaActionDecorator`?
Adding a small class documentation would also help, and a small test for the bind method would also be nice.

Reply:
updated!

Reply:
added the test as well!